api/src/main/java/org/apache/iceberg/io/CloseableIterable.java

```diff
@@ -53,8 +54,18 @@ public void close() {
     };
   }
 
-  static <E> CloseableIterable<E> combine(Iterable<E> iterable, Iterable<Closeable> closeables) {
-    return new CloseableGroup.ClosingIterable<>(iterable, closeables);
+  static <E> CloseableIterable<E> combine(Iterable<E> iterable, Closeable closeable) {
```

Comment:
It seems `combine` can be implemented as `wrap` with an `Identity` function?

Reply:
hmm, maybe not. I see their signatures are different.

Reply:
Wrap takes a CloseableIterable and a function to transform the iterable itself, like bin packing. Combine takes an iterable and associates it with something to close. I think that `wrap` could be implemented using `combine` instead. I'll update this and remove `wrap`.

Reply:
Fixed.